src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -477,6 +933,23 @@ void LSHSearch<SortPolicy>::Search(const arma::mat& querySet,
   if (k == 0)
     return;
 
+  // If the user requested more than the available number of additional probing
+  // bins, set Teffective to maximum T. Maximum T is 2^numProj - 1
+  size_t Teffective = T;
+  if (T > ( (size_t) ( (1 << numProj) - 1) ) )
+  {
+    Teffective = ( 1 << numProj ) - 1;
+    Log::Warn << "Requested " << T << 
+      " additional bins are more than theoretical maximum. Using " <<
+      Teffective << " instead." << std::endl;
```

Comment:
I like to make these lines as long as I can by splitting the strings up, and starting new lines with `<<`.  Whatever you want to do is fine, but the new lines should at least be indented four characters instead of two.